### ISSUE TITLE
Create a component that allow adding aria-labels

### ISSUE THREAD

**Issue by kyrapieterse** [maxi-blocks/maxi-blocks#5112]
### Discussed in https://github.com/maxi-blocks/maxi-blocks/discussions/4845

<div type='discussions-op-text'>

<sup>Originally posted by **ckp267** May 12, 2023</sup>
**From Josh**
I'd like to share this with you and let you choose the best option. Kyra requested yesterday to check the score and rank Google Lighthouse was giving for a page; found out that accessibility wasn't reaching all the score we'd like, and one of the points is related with the aria-labels. Been reading about it and found a really interesting link => https://www.w3.org/WAI/ARIA/apg
I understand you aren't going to read right now, so giving a bit of resume: ARIA or  Accessible Rich Internet Application is used to generate accessible experience in the web. The first suggestion is No ARIA is better than Bad ARIA., so before we start using it we should be cautious with it. Then they have this other page which I think is incredible and will make our job easier: https://www.w3.org/WAI/ARIA/apg/patterns/. They share how to implement this aria-labels and what they need. So, here's the situation I'd like you to let me know what you prefer:
1. I can do some fast fix for the links to score correctly on the Google Lighthouse, which will be faster but will have some risks ( No ARIA is better than Bad ARIA.) and will affect just links, which means we can test another page on Google Lighthouse and say error of accessibility in other blocks.
**2. I can take a bit longer and create a component that allow adding aria-labels to creators and ensure a whole control of the accessibility to have an easier way to score 100.**

What do you think is better with the current context? Don't hesitate on asking what you need 🙏🙂

**Kyra Pieterse, Yesterday 11:28**
option 2 sounds better but lets see what C says

**Joshua Juan Caparrós, Yesterday 11:28**
cool, I don't think second option will take me longer than 2 or 3 days and will ensure a good accesibility feature 🙂

**Christiaan Pieterse (yc), 8 min**

Thank you for the info @Joshua Juan Caparrós yeah, option 2 seems like a better way to go long-term. The important thing is that it's not a quick fix - it should be included in all planning and thinking. It's easier to add it in from the beginning than to try to fix it later.  If we add it as a component then we can help make it easier for people to do it themselves. That makes sense.
More ideas here.
https://theplusblocks.com/docs/aria-labels-in-wordpress/ 
https://fullsiteediting.com/lessons/accessibility-in-full-site-editing-themes/

Joshua Juan Caparrós, 7 min

thanks for the answer </div>
